src/ucp/wireup/address.c

```diff
@@ -766,8 +768,11 @@ static int ucp_address_pack_iface_attr(ucp_worker_h worker, void *ptr,
          * depends on device NUMA locality. */
         unified            = ptr;
         unified->rsc_index = rsc_index;
-        unified->lat_ovh   = enable_atomics ? -iface_attr->latency.c :
-                                               iface_attr->latency.c;
+        lat_ovh            = ucp_wireup_iface_lat_distance_v1(wiface);
```

Comment:
shouldn't we check v1/v2 here?

Reply:
Do you mean checking address version to decide whether we should use `ucp_wireup_iface_lat_distance_v1` or `ucp_wireup_iface_lat_distance_v2`?

Reply:
ok